### DISCUSSION TITLE
Measuring VCC voltage

### DISCUSSION THREAD

**Discussion by metehoca** [SpenceKonde/megaTinyCore#1181] [Q&A]
I'm using an ATtiny3216 (or ATtiny816) with a 1V2 external analog reference. Is there a way to measure the MCU's supply voltage without using any ADC pins?

EDIT: My bad; https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/extras/Ref_Analog.md#measuring-vdd-on-01-series

**Comment by metehoca** [comment]
I get "'ADC_INT_REF' was not declared in this scope" error.

**Comment by jvasileff** [comment]
The example code uses `ADC_INTREF`

https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/libraries/megaTinyCore/examples/readTempVcc/readTempVcc.ino
